pkg/provider/resource_nrs_alert_condition.go

```
@@ -79,6 +84,23 @@ func NRSAlertConditionCreate(resourceData *schema.ResourceData, meta interface{}
 	return nil
 }
 
+
+func NRSAlertConditionImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+    s := strings.Split(d.Id(), ":")
+    if len(s) != 2 {
+        /*
+            In New Relic alert condition , we need both policy_id  and condition_id to get given user data.
+        */
+        return nil, fmt.Errorf("Import resource ID should consist of policy_id:condition_id")
+    }
+
+    val,_ := strconv.Atoi(s[0])
```

Although this failing is improbable, let's not ignore the error.